doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+### Scatter chart 
+
+The $PlatformShort$ Scatter Chart, or Scatter Graph, is used to show the relationship between two values using a Cartesian (X, Y) coordinate system to plot data. Each data point is rendered as the intersecting point of the data value on the X and Y Axis. Scatter charts draw attention to uneven intervals or clusters of data. They can highlight the deviation of collected data from predicted results and they are often used to plot scientific and statistical data. The $PlatformShort$ Scatter chart organizes and plots data chronologically (even if the data is not in chronological order before binding) on X-Axis and Y-Axis. Learn more about our [scatter charts](chart-types-scatter.md).
+
+### Spline Chart 
+
+The $PlatformShort$ Spline Chart, or Spline Graph is a type of category line graph shows the continuous data values represented by points connected by smooth line segments of one or more quantities over a period time for showing trends and performing comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) are showing a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart. The $PlatformShort$ Spline chart is identical to the $PlatformShort$ Spline chart, the only different being the line chart is points connected by straight lines, and the spline chart points are connected by smooth curves. Learn more about our [spline charts](chart-types-spline.md).
+
+### Stepline Chart 
+
+A Step Line Chart, or Step Line Graph, is a category charts that renders a collection of data points connected by continuous vertical and horizontal lines forming a step-like progression. Values are represented on the Y-Axis (left labels) and categories are displayed on the X-Axis (bottom labels). The $PlatformShort$ Step Line chart emphasizes the amount of change over a period of time or compares multiple items. The $PlatformShort$ Step Line chart is identical to the $PlatformShort$ step area chart in all aspects except that the area below the step lines is not filled in. Learn more about our [step line charts](chart-types-step.md)
+
+### Timeline Charts / Time-Series Charts 
+
+A Time-Series Chart, or Timeline Graph, is a visualization that treats the data as a sequence of category data items that are sorted by then rendered by date. Labels on this axis are placed along the X-Axis (bottom Axis), according to the date-time value. The $PlatformShort$ Time-Series is use to show Financial Series, Range Series, and Category Series (Line, Area, Column, Point, Spline, Scatter, Waterfall and the Stacked equivalents of those chart types). The Time-Series also supports the ability to exclude intervals of data with breaks. As a result, labels will not appear at the excluded interval. For example, working/non-working days, holidays, or weekends.
+
+### Treemap
+
+Learn more about our [treemaps](../treemap-overview.md).
+
```

Comment:
This is Andie's design for the treemap chart:
https://invisionapp.swoogo.com/servicedesigndc-squads?utm_campaign=VIR_Squads%20Service%20Design%20DC%20Partner%20Screening_02-11-21&utm_source=marketo&utm_medium=email&utm_content=Cta&mkt_tok=eyJpIjoiTldRMk1UUTBNVGt3TkRRdyIsInQiOiJxaGNuRUQzOTBFV0VYTmpENVNRM1VKRCtcLzg2b2g0b3hQK256WFNzc01UNDZzNlJ0Q1N6TnRiWkNmOHJ2eVc0UTRHSUV3cEI1b0hKM3ZFQzJ5N1F1V1ZSVERTMVlkSFdiekZJR3JtZERWZ0JQdmszNjJmY0pHWTdVeVB6ZUtuZzcifQ%3D%3D

Reply:
Yes just disregard the palettes she uses because there are screens describing them and use the ones @tosharky provided in his other files

Reply:
I've actually changed the colors to match the palette.

Reply:
I worte this because first thing on the link was a palette and then when I went to other pages I saw design, so I wanted to point out to just skip the palette as we are not implying the use of custom one, but want to have the standard one across all samples